src/client/assets/css/fonts.scss

```diff
@@ -0,0 +1,9 @@
+@font-face {
+  font-family: 'OpenSans';
+  src: url('/fonts/OpenSans-Regular.eot'); /* IE9 Compat Modes */
+  src: url('/fonts/OpenSans-Regular?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('/fonts/OpenSans-Regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('/fonts/OpenSans-Regular.woff') format('woff'), /* Pretty Modern Browsers */
+       url('/fonts/OpenSans-Regular.ttf')  format('truetype'), /* Safari, Android, iOS */
+       url('/fonts/OpenSans-Regular.svg#svgOpenSans-Regular') format('svg'); /* Legacy iOS */
```

Comment:
Yeah this doesn't work at all - Turns out Noah had the font installed so it appeared to be working. Closing for now -- new PR up this afternoon.